### DISCUSSION TITLE
run podman-compose on a remote machine

### DISCUSSION THREAD

**Discussion by youone** [containers/podman-compose#902] [Q&A]
Can you run podman-compose locally with a local podman-compose.yml and have it spin up on an other machine? 

Something with the same functionality as adding "--remote --connection host" flags in the podman command

**Comment by eleloi** [comment]
You can set the environment variable `CONTAINER_HOST=ssh... or unix...` and then podman runs in remote mode whith this host by default, without the flag, and compose works too

**Reply by bam80** [reply]
The env var is fine, but still, an option would be useful, in case you can't or don't want to use the var.
And CONTAINER_HOST requires full URI and doesn't accept the connection name as in `--connection host`.
